src/core.js

```diff
@@ -272,6 +272,16 @@ $.extend( $.validator, {
 				this.element( element );
 			}
 		},
+		oninput: function( element ) {
+			if ( element.name in this.submitted || element.name in this.invalid ) {
+				this.element( element );
+			} else {
+				return;
+			}
+		},
+
+		// DEPRECATED
+		// Please use `oninput` instead.
 		onkeyup: function( element, event ) {
```

Comment:
> So as I understand it, this will also add validation e.g. after paste content into a input. We have to test carefully.

Correct.

>> So the user now is supposed to use the 2 handlers to support legacy browsers (even though I think it's a waste of time trying to support <IE9 and other legacy browsers)

> We need to document that

I'll send a PR to document the `oninput` handler asap, and sure I'll mention this.